ios/brave-ios/Sources/AIChat/Resources/tr.lproj/BraveLeo.strings

```diff
@@ -308,7 +353,7 @@
 
 /* The description terms and conditions for AI chat usage. The links are inside parenthesis %@ will be replaced with urls */
-"aichat.termsConditionsDescription" = "Brave Leo, internet kullanımınızı geliştiren özel bir akıllı Yapay Zeka asistanıdır. Leo'nun kısıtlı erişimle kullanımı ücretsizdir. Brave Leo Premium daha fazla model ve daha yüksek limitler sunar ve abonelere yeni özelliklere erken erişim imkanı sunar. Tüm kullanıcılar için varsayılan model şu anda Mixtral 8x7B'dir. Daha fazla detay için **[Brave wiki](%1$@)** ye bakın.\n\nVarsayılan olarak, Leo'ya bir soru sorduğunuzda, Brave Leo size yanıt vermek için görüntülemekte olduğunuz web sayfasının içeriğini veya bir sayfada bulunan vurguladığınız herhangi bir metni gönderebilir ve kullanabilir. Yanıtların doğruluğu garanti edilmez ve hatalı, yanlış yönlendiren bilgiler içerebilir. Leo, yanıt kalitesini geliştirmek için Brave Search'ten edindiği verileri kullanmaktadır. Hassas veya özel bilgileri göndermeyin ve sağlık, finans, kişisel güvenlik veya benzeri konularla ilgili yanıtları değerlendirirken dikkatli olun. Leo'nun seçeneklerini Ayarlar bölümünden istediğiniz zaman değiştirebilirsiniz. Ayarlar ve Leo bölümüne gidin.\n\nLeo, IP adresi gibi, sizinle ilişkilendirilebilecek tanımlayıcı bilgileri toplamaz. Yapay Zeka modellerini eğitmek için hiçbir veri kullanılmaz ve Yapay Zeka modelleri (verileri silmeden önce 30 gün boyunca tutan Anthropic dışında) hiçbir kişisel veriyi tutmaz. Daha fazla bilgi için **[gizlilik politikasına](%2$@)** bakın.";
+"aichat.termsConditionsDescription" = "Brave Leo, web kullanımınızı geliştiren özel bir YZ asistanıdır. Leo'yu kullanmak, sınırlı erişimle ücretsizdir. Brave Leo Premium ise daha fazla model ve daha yüksek limitler sunar, ayrıca abonelerin yeni özellikleri erkenden deneyimlemesine olanak tanır. Şu anda tüm kullanıcılar için varsayılan model Mixtral 8x7B'dir. Daha fazla bilgi için Brave **[Yardım Merkezi](%1$@)**'ne bakın.\n\nVarsayılan olarak, Leo'ya bir soru sorduğunuzda Leo görüntülediğiniz web sayfasının içeriğini veya sayfada, Brave Leo'nun yanıt vermesini istediğiniz için vurguladığınız metinleri gönderebilir ve kullanabilir. Bazı durumlarda, Leo Brave Arama'ya girdiğiniz sözcükler kullanılarak türetilen sorgular gönderir ve sorunuzu daha iyi yanıtlayabilmek için arama sonuçlarını kullanır. Yanıtların doğruluğu garanti edilmez ve hatalı, yanlış yönlendiren veya yanlış bilgiler içerebilir. Hassas ve gizli bilgiler göndermeyin ve sağlık, finans, kişisel güvenlik veya benzeri cevapları kullanırken dikkatli olun. Leo'nun seçeneklerini Ayarlar menüsünden istediğiniz zaman değiştirebilirsiniz. Ayarlar > Leo bölümüne gidin.\n\nLeo, IP adresi gibi, sizinle ilişkilendirilebilecek tanımlayıcı bilgileri toplamaz. YZ modellerini eğitmek için hiçbir veri kullanılmaz ve YZ modelleri (verileri silmeden önce 30 gün boyunca tutan Anthropic dışında) hiçbir kişisel veriyi tutmaz. Daha fazla bilgi için **[gizlilik politikasına](%2$@)** bakın.";
```

Comment:
"aichat.termsConditionsDescription" = "Brave Leo, web kullanımınızı geliştiren özel bir Yapay Zeka asistanıdır. Leo'yu kullanmak, sınırlı erişimle ücretsizdir. Brave Leo Premium ise daha fazla model ve daha yüksek limitler sunar, ayrıca abonelerin yeni özellikleri erkenden deneyimlemesine olanak tanır. Şu anda tüm kullanıcılar için varsayılan model Mixtral 8x7B'dir. Daha fazla bilgi için Brave **[Yardım Merkezi](%1$@)**'ne bakın.\n\nVarsayılan olarak, Leo'ya bir soru sorduğunuzda Leo görüntülediğiniz web sayfasının içeriğini veya sayfada, Brave Leo'nun yanıt vermesini istediğiniz için vurguladığınız metinleri gönderebilir ve kullanabilir. Bazı durumlarda, Leo Brave Arama'ya girdiğiniz sözcükler kullanılarak türetilen sorgular gönderir ve sorunuzu daha iyi yanıtlayabilmek için arama sonuçlarını kullanır. Yanıtların doğruluğu garanti edilmez ve hatalı, yanlış yönlendiren veya yanlış bilgiler içerebilir. Hassas ve gizli bilgiler göndermeyin ve sağlık, finans, kişisel güvenlik veya benzeri cevapları kullanırken dikkatli olun. Leo'nun seçeneklerini Ayarlar menüsünden istediğiniz zaman değiştirebilirsiniz. Ayarlar > Leo bölümüne gidin.\n\nLeo, IP adresi gibi, sizinle ilişkilendirilebilecek tanımlayıcı bilgileri toplamaz. Yapay Zeka modellerini eğitmek için hiçbir veri kullanılmaz ve Yapay Zeka modelleri (verileri silmeden önce 30 gün boyunca tutan Anthropic dışında) hiçbir kişisel veriyi tutmaz. Daha fazla bilgi için **[gizlilik politikasına](%2$@)** bakın.";